deployment/roles/pycapa/tasks/dependencies.yml

```diff
@@ -15,20 +15,22 @@
 #  limitations under the License.
 #
 ---
-- name: Install epel-release repository
-  yum: name=epel-release
+- name: Install yum repositories
+  yum: name={{ item }} update_cache=yes
+  with_items:
+    - epel-release
+    - centos-release-scl
 
-- name: Install python and tcpdump
-  yum:
-    name: "{{item}}"
+- name: Install dependencies
+  yum: name={{ item }}
   with_items:
-    - python
-    - tcpdump
-    - git
+    - "@Development tools"
+    - python27
```

Comment:
Yes, this installs python 2.7 at a different location and does not change the default 2.6 install.